MvvmCross-Forms/MvvmCross.Forms.Droid/Views/MvxFormsAndroidViewPresenter.cs

```diff
@@ -46,7 +45,7 @@ public virtual IMvxFormsPagePresenter FormsPagePresenter
                     _formsPagePresenter = new MvxFormsPagePresenter(FormsApplication, ViewsContainer, ViewModelTypeFinder, attributeTypesToActionsDictionary: AttributeTypesToActionsDictionary);
                     _formsPagePresenter.ClosePlatformViews = ClosePlatformViews;
                     _formsPagePresenter.ShowPlatformHost = ShowPlatformHost;
-                    Mvx.RegisterSingleton<IMvxFormsPagePresenter>(_formsPagePresenter);
+                    Mvx.RegisterSingleton(_formsPagePresenter);
```

Comment:
I think you need to revert this

Reply:
Doh! sorry my bad, of course this registers it with the wrong key. Will do

Reply:
_formsPagePresenter is only of type IMvxFormsPagePresenter so we don't need to specify the type. This change should be able to stay